### DISCUSSION TITLE
venia implementation

### DISCUSSION THREAD

**Discussion by janbiesiada** [Smile-SA/elasticsuite#3164] [Q&A]
I'm looking for solution that allows website on pwa-studio and elasticsuite with working search analysis

**Comment by romainruaud** [comment]
what do you mean by "working search analysis" ?

**Reply by janbiesiada** [reply]
the "Search Usage Analytics" - it does't show me any results after search - I'm able to force it using tracking pixel or api endpoint so the indexes and setup are correct.
